docs/usage.rst

```diff
@@ -44,6 +44,54 @@ Command-Line Arguments
    :prog: xcp_d
 
 
+.. _filter_files:
+
+Filtering Inputs with BIDS Filter Files
+=======================================
+
+``xcp_d`` allows users to choose which preprocessed files will be post-processed with the ``--bids-filter-file`` parameter.
+This argument must point to a JSON file, containing filters that will be fed into PyBIDS.
+
+The keys in this JSON file are unique to ``xcp_d``.
+They are our internal terms for different inputs that will be selected from the preprocessed dataset.
+
+``"bold"`` determines which preprocessed BOLD files will be chosen.
+You can set a number of entities here, including "session", "task", "space", "resolution", and "density".
+We recommend NOT setting the datatype, suffix, or file extension in the filter file.
```

Comment:
Maybe we can add a warning block that says we don't recommend adding additional filters to any of the below fields?

Reply:
Added in [7ae3994](https://github.com/PennLINC/xcp_d/pull/686/commits/7ae3994fe83698861feb242635463e1dbd7cdaad)